packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2507,26 +2505,27 @@ test('manager notifiers', async t => {
       Collateral: t.context.aeth.mint.mintPayment(aeth.make(AMPLE)),
     }),
   );
-  const { vault: vault1 } = await E(vaultSeat).getOfferResult();
+  let { vault } = await E(vaultSeat).getOfferResult();
```

Comment:
I'm not a fan of re-using variables in this way, but if that's the way this test goes, please update vault2 (line 2726) to be part of it.

Reply:
Not my favorite either. I think this would be better with a driver but that's more work that I can't justify. At one point I had them numbered, but that made it harder to move things around. 

(done as a fixup and rebase)